modules/top-level/plugins/default.nix

```diff
@@ -0,0 +1,52 @@
+/*
+  Creates the final list of (normalized) plugins that will be passed to the wrapper.
+
+  It notably implements:
+  - byte-compilation (performance.byteCompileLua): in this file
+  - plugins combining (performance.combinePlugins): in ./combine-plugins.nix
+*/
+{
+  lib,
+  config,
+  pkgs,
+  builders,
+  ...
+}:
```

Comment:
Initially, I'd expect top-level/plugins to be a module. Maybe we can imply a more functional name (verb), something like `normalise-plugins`?

If it is a function, we should avoid `...` to keep the arguments explicit.

---

If it is possible to implement as a module, I would prefer that.

Couldn't we just have an internal `normalisedPlugins` option that this module defines?